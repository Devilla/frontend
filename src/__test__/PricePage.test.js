// import React from 'react';
// import { shallow, mount } from 'enzyme';
// import renderer from 'react-test-renderer'
// import configureStore from 'redux-mock-store'
// import { Provider } from 'react-redux'
// import {createStore} from 'redux'
// import { load, loaded } from 'ducks/loading';
// import { PricePage } from 'components';
// import { store } from 'App';
//
// // Snapshot for Home React Component
// describe('>>> PricePage --- Snapshot',()=>{
//     it('+++capturing Snapshot of PricePage', () => {
//         const renderedValue =  renderer.create(<PricePage store={store} />).toJSON()
//         expect(renderedValue).toMatchSnapshot();
//     });
// });
