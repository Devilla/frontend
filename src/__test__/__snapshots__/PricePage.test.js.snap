// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> PricePage --- Snapshot +++capturing Snapshot of PricePage 1`] = `
<div
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <div
      className="price"
    >
      <div
        className="pricing-row w-row"
        style={
          Object {
            "width": "100%",
          }
        }
      >
        <div
          className="w-col"
        >
          <ul>
            <li>
              <span
                className="btn btn-outline-info waves-light waves-effect mr-2 set-br"
                onClick={[Function]}
              >
                Monthly
              </span>
            </li>
            <li
              className="mt-3"
              style={
                Object {
                  "display": "none",
                }
              }
            >
              <span
                className="react-flexible-switch react-flexible-switch--active"
                onMouseLeave={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "border": "1px solid #CFCFCF",
                    "borderRadius": 25,
                    "boxSizing": "content-box",
                    "display": "flex",
                    "padding": 4,
                    "position": "relative",
                    "width": 50,
                  }
                }
              >
                <span
                  className="react-flexible-switch-label"
                  style={
                    Object {
                      "left": "20% ",
                      "pointerEvents": "none",
                      "position": "absolute",
                      "top": "50%",
                      "transform": "translateY(-50%)",
                    }
                  }
                >
                  
                </span>
                <span
                  className="react-flexible-switch-circle"
                  style={
                    Object {
                      "backgroundColor": "#097fff",
                      "borderRadius": 35,
                      "diameter": 18,
                      "display": "block",
                      "height": 18,
                      "offColor": "grey",
                      "onColor": "#097fff",
                      "transform": "translateX(32px)",
                      "transition": "transform 200ms, width 200ms, background-color 200ms",
                      "width": 18,
                    }
                  }
                />
                <button
                  disabled={false}
                  onClick={[Function]}
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "borderColor": "transparent",
                      "color": "transparent",
                      "height": "100%",
                      "left": 0,
                      "pointerEvents": "none",
                      "position": "absolute",
                      "top": 0,
                      "width": "100%",
                    }
                  }
                  type="button"
                />
              </span>
            </li>
            <li>
              <span
                className="active btn btn-outline-success waves-light waves-effect ml-2 set-br"
                onClick={[Function]}
              >
                 Yearly 
              </span>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="row"
    />
  </div>
</div>
`;
